curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-tech-trends-and-updates/66360c141117e030da0c0cb3.md

```diff
@@ -0,0 +1,39 @@
+---
+id: 66360c141117e030da0c0cb3
+title: Task 12
+challengeType: 22
+dashedName: task-12
+---
+
+<!--
+AUDIO REFERENCE: 
+Sarah: It's particularly useful when you have many tasks and a limited coding team.
+-->
+
+# --description--
+
+The adverb `particularly` is used to emphasize that something is especially, or more than usually, the case in certain context. It's similar to saying `especially` or `notably`.
+
+For example, `This tool is particularly helpful for beginners.` This means it's especially helpful for beginners more than others.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`It's _ useful _ you have many tasks and a limited coding team.`
+
+## --blanks--
+
+`particularly`
+
+### --feedback--
+
+This word is used to emphasize the greater usefulness of something under specific conditions.
```

Comment:
I know `when` is a good word here, but will English learners have an easy time realizing that it's that word and not something like `for` or `if`.  The word `for` is a stretch but it was mentioned in the description. 

Giving it a second thought, is `if` a better word here than `when`?

Reply:
This task asks learners to listen to a dialogue and then fill in the blanks. The word mentioned in the dialogue is `when`.

`When` was taught in previous task, and this task is asking the learners to practice using it. Some tasks are easier to complete, so learners can gain confidence.

Reply:
Yea, I think we should stick with `when` here. There will be an animation that reads the sentence using `when`, so changing it to something else would probably be confusing.